collectors/ebpf.plugin/README.md

```diff
@@ -84,27 +159,25 @@ The Agent counts and shows the number of instances where a running program exper
 
 #### Create
 
-This chart shows the number of calls for `vfs_create`. This function is responsible to create files.
+This chart shows the number of calls to `vfs_create`. This function is responsible for creating files.
 
 #### Synchronization
 
-This chart shows the number of calls for `vfs_fsync`. This function is responsible to perform a fsync or fdatasync 
-on a file.
+This chart shows the number of calls to `vfs_fsync`. This function is responsible for performing a fsync or fdatasync 
+on a file. You can see more details on Synchronization
```

Comment:
Would be good to link this to the line 90 (#syncrhonization) but I'm not sure how this will work, since we have two sub-sections with Synchronization name...

Reply:
I agree, let us merge this code like it is and when I bring more updates with other PRs, we create the links.